apis/openstack-swift/src/test/java/org/jclouds/openstack/swift/v1/blobstore/RegionScopedSwiftBlobStoreParallelLiveTest.java

```diff
@@ -121,7 +126,7 @@ public void uploadMultipartBlob() {
 
    @Test(dependsOnMethods = "uploadMultipartBlob", singleThreaded = true)
    public void downloadParallelBlob() throws IOException {
-      final File downloadedFile = new File(BIG_FILE.getName() + ".downloaded");
+      final File downloadedFile = new File(BIG_FILE + ".downloaded");
```

Comment:
Yes, sorry, my comment wasn't clear: I was wondering whether there's a risk here - even with an absolute path - that two test runs would write the same file. But I don't think that's the case.